app/services/movie_data_service.rb

```diff
@@ -249,56 +249,5 @@ def get_common_movies_between_multiple_actors(actor_names: nil, paginate_actor_n
         total_pages: movie_response[:total_pages]
       )
     end
-
-    def get_movie_streaming_service_providers(movie)
```

Comment:
This is replaced by a slightly more robust version that can also handle requests for `tv` data.